ignite/config/plugins/config.go

```diff
@@ -16,7 +16,9 @@ type Config struct {
 	// path to the config file
 	path string
 
-	Plugins []Plugin `yaml:"plugins"`
+	// Apps holds the list of installed Ignite Apps.
+	// Ignite Apps are implemented as plugins.
+	Apps []Plugin `yaml:"apps"`
```

Comment:
Can the type be rename as well? Or is that a choice to keep the naming?

We could always alias Plugin to it and deprecate it.

Reply:
The idea of the PR is to only change the user facing parts from plugin to Ignite Apps. The reason for that is that the name apps fits better with the plans moving forward. Soon we will publish a blog article which will add more context to these changes.

Refactoring the code is something we have to discuss first because it will take more time and will impact other repositories.

Reply:
Alright! I missed that context. Thanks for the clarification!